### DISCUSSION TITLE
LogEntryTags not being created

### DISCUSSION THREAD

**Discussion by xanadontrad** [jongpie/NebulaLogger#815] [Q&A]
Hi there! I have Nebula installed in my production org and sandboxes. I am testing out Logger Tag Rules configured in the custom metadata in a sandbox, but despite several tests, no Log Entry Tag records are being created for my Log Entries. (They work in my developer pro sandbox, but not my full copy) Is there some configuration I am missing or a way I can troubleshoot?

Thanks in advance!! (Added screenshots in comments)

**Comment by xanadontrad** [comment]
Screenshots of settings and an example entry (I have created several dozen while troubleshooting).

**Logging sObject Handler Config:**
![image](https://github.com/user-attachments/assets/63103b3f-9c7f-483b-902b-38aac4240c6b)

**Logging Parameters:**
![image](https://github.com/user-attachments/assets/dbfe5ac6-606b-456c-9d88-d70b43fbeb8a)

**Logging Tag Rule:**
![image](https://github.com/user-attachments/assets/ddc72052-aedd-4576-9f4e-5fd8b2a95123)

**Logger Tags:**
![image](https://github.com/user-attachments/assets/4e38ac9c-a98e-49de-a54f-5cf6f56383b5)

**Example Log Entry:**
![image](https://github.com/user-attachments/assets/72ec4144-c12c-4163-8997-561029359957)

**Reply by jongpie** [reply]
@xanadontrad thanks for providing all of those screenshots, they're incredibly helpful! Offhand, I don't see any issues with your configuration - everything looks correct in your screenshots. I'll have to do some testing to see if I can reproduce the issue/see what's going on. Do you know which version of Nebula Logger you have installed in your orgs? I want to make sure I'm testing with the same version, just in case it's a bug in certain version(s).

**Reply by xanadontrad** [reply]
Thanks, Jonathan! We have version 4.11 installed in both the sandboxes.
